lib/service/Service.ts

```diff
@@ -325,8 +325,8 @@ class Service extends EventEmitter {
   /**
    * resolveHash resolve hash to preimage.
    */
-  public resolveHash = async (args: { hash: string }) => {
-    return this.swaps.resolveHash(args);
+  public resolveHash = async (request: lndrpc.ResolveRequest) => {
+    return this.swaps.resolveHash(request);
```

Comment:
Also doesn't have to be in this PR, but it would be good to get some argument checks in here - like amount has to be > 0 and hash should match a certain length, etc...

Reply:
hash is check  - if not valid you will not find the deal.

the amount is checked - if not valid it will not match to the deal amount.

Same for the other 2 values. If these are wrong we will catch it by one of the checks.

I see no need to enhance it (until we see an example of such a need).

Reply:
Basically I just mean to return a [INVALID_ARGUMENT] error if the argument is just completely invalid. Not for this PR, just a thought.